etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -42,6 +42,7 @@ var (
 	ErrGRPCRootUserNotExist     = grpc.Errorf(codes.FailedPrecondition, "etcdserver: root user does not exist")
 	ErrGRPCRootRoleNotExist     = grpc.Errorf(codes.FailedPrecondition, "etcdserver: root user does not have root role")
 	ErrGRPCUserAlreadyExist     = grpc.Errorf(codes.FailedPrecondition, "etcdserver: user name already exists")
+	ErrGRPCUsernameEmpty        = grpc.Errorf(codes.InvalidArgument, "etcdserver: username is empty")
```

Comment:
s/UsernameEmpty/UserEmpty and s/username/user name/ to be consistent with related error messages